src/org.xtuml.bp.core/src/org/xtuml/bp/core/common/NonRootModelElement.java

```diff
@@ -773,6 +773,11 @@ public boolean loadProxy() {
 				&& (!getModelRoot().getId().equals(
 						Ooaofooa.COMPARE_MODEL_ROOT_NAME) && !getModelRoot().isCompareRoot())) {
             result = PersistenceManager.loadAndFinishComponent(m_contentPath);
+            if(!result) {
```

Comment:
No, the calls to load proxy are coming from already loaded RGOs.  In most cases this batch relate will never be called as the RGO will already have had batchRelate called once.  This is specifically for the case where the RTO is in an inaccessible location during the RGO load.  If later a traversal is made from the RGO to such an RTO, this batchRelate would then be called and would locate the RTO.